PHPCompatibility/Tests/Sniffs/PHP/ArgumentFunctionsReportCurrentValueSniffTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Argument Functions Report Current Value sniff test file
+ *
+ * @package PHPCompatibility
+ */
+
+namespace PHPCompatibility\Tests\Sniffs\PHP;
+
+use PHPCompatibility\Tests\BaseSniffTest;
+
+/**
+ * Argument Functions Report Current Value sniff tests
+ *
+ * @group deprecatedFunctions
+ * @group functions
+ *
+ * @covers \PHPCompatibility\Sniffs\PHP\DeprecatedFunctionsSniff
+ *
+ * @uses    \PHPCompatibility\Tests\BaseSniffTest
+ * @package PHPCompatibility
+ * @author  Wim Godden <wim.godden@cu.be>
+ */
+class ArgumentFunctionsReportCurrentValueSniffTest extends BaseSniffTest
+{
+
+    const TEST_FILE = 'sniff-examples/argument_functions_report_current_value.php';
+
+
+    /**
+     * testDeprecatedFunction
+     *
+     * @dataProvider dataFunction
+     *
+     * @param string $functionName      Name of the function.
+     * @param string $deprecatedIn      The PHP version in which the function was deprecated.
+     * @param array  $lines             The line numbers in the test file which apply to this function.
+     * @param string $okVersion         A PHP version in which the function was still valid.
+     * @param string $deprecatedVersion Optional PHP version to test deprecation message with -
+     *                                  if different from the $deprecatedIn version.
+     *
+     * @return void
+     */
+    public function testFunction($functionName, $deprecatedIn, $lines, $okVersion, $deprecatedVersion = null)
+    {
+        $file = $this->sniffFile(self::TEST_FILE, $okVersion);
```

Comment:
I would remove this block in favour of a `testNoViolationInFileOnValidVersion()` test like we have in most other sniff test files.